### ISSUE TITLE
Call to undefined function imagecreate()

### ISSUE THREAD

**Issue by deric** [Mikesch-mp/icingaweb2-module-grafana#276]
```
Call to undefined function Icinga\Module\Grafana\Controllers\imagecreate() in /usr/share/icingaweb2/modules/grafana/application/controllers/ImgController.php:217
```
-->

## Expected Behavior
Expected to see a graph

## Possible Solution

It might be worth mentioning in installation guide that the PHP GD module is required. E.g. on Debian:
```
apt install php7.3-gd
```

## Your Environment

* Icinga Web 2 Grafana version (System - About):
* Icinga Web 2 version and modules (System - About):
* Version used (`icinga2 --version`): `r2.12.5-1`
* Operating System and version:
* Enabled features (`icinga2 feature list`): ` api checker ido-pgsql influxdb mainlog notification`
* Config validation (`icinga2 daemon -C`):

**Comment by v0tti** [comment]
I ran into the same problem

**Comment by Mikesch-mp** [comment]
Fix in Documentation for 2.0.0
